views/plugin/featureaction/featuresoundactionconfig.html

```diff
@@ -0,0 +1,19 @@
+<div class="feature-action-sound-config">
+  <select
```

Comment:
This template is not following the Bootstrap 4 design decisions:
- Don't add classes (like `feature-action-sound-config`) that aren't used anywhere. Classes should be prefixed with `c-` if they have SCSS to style them, and `js-` if they're used in a JS selector.
- The select should have `class="form-control custom-select"`, and a label (`Sound`).
- The table is overkill and should be a `div.form-group` with a label and the spinner control. You can use `col-*` classes to align the labels/controls for the sound and delay.
- Form labels should not have a `:` after them, and should be right-justified (`text-right` class).
- Avoid using `style`. You can use `w-100` or `col` (depending on alignment with other controls) to get a full-width control.

We still need to write all of this up in the repo, but for now you can look through existing forms and Bootstrap 4 docs for additional guidance.

Reply:
These changes are current with the last commit , except for the tooltip implementation